### PR TITLE
Add Ecuador holidays

### PR DESCRIPTION
**Pull request by KJhellico**
## Proposed change

Add Ecuador holidays (en_US, es, uk localization).

## Type of change

- [x] New country/market holidays support (thank you!)
- [ ] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency upgrade (version update)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good (`make pre-commit`)
- [x] I've added tests to verify that the new code works and all tests pass locally (`make test`, `make tox`)
- [x] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/dr-prodigy/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/dr-prodigy/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/countries/ecuador.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                if obs_date and obs_date not in observed_dates:
+                    self._add_holiday(
+                        self.tr("%s (Observado)") % self[dt], obs_date
+                    )
+
+            dt = date(year, DEC, 31)
+            if self._is_friday(dt):
+                self._add_holiday(
+                    self.tr("%s (Observado)") % self.tr("Año Nuevo"), dt
+                )
```
Comment (arkid15r) [inline holidays/countries/ecuador.py:122]:
```suggestion
            dec_31 = date(year, DEC, 31)
            if self._is_friday(dec_31):
                self._add_holiday(
                    self.tr("%s (Observado)") % self.tr("Año Nuevo"), dec_31
                )
```

File: tests/countries/test_ecuador.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+        run_tests((EC.default_language, None, "invalid"))
+
+        self.set_language("en_US")
+        run_tests((EC.default_language,))
+
+    def test_l10n_en_us(self):
+        en_us = "en_US"
+
+        cnt = EC(language=en_us)
```
Comment (arkid15r) [inline tests/countries/test_ecuador.py:164]: What's `cnt`?

**Review by arkid15r — COMMENTED** [review]
\+ 🇪🇨 
Nicely done! No `self.assertHolidays()` though :(
LGTM!

File: tests/countries/test_ecuador.py (hunk 2 above)
Comment (KJhellico) [reply tests/countries/test_ecuador.py:164]: Country :)

File: holidays/countries/ecuador.py (hunk 1 above)
Comment (KJhellico) [reply holidays/countries/ecuador.py]: Done.

**Pull request merged**